src/groundlight/encodings.py

```diff
@@ -29,12 +28,12 @@ def url_encode_dict(maybe_dict: Union[Dict, str], name: str, size_limit_bytes: O
             raise TypeError(f"`{name}` must be a dictionary or JSON string: got {original_type}") from e
 
     if not isinstance(maybe_dict, dict):
-        raise TypeError(f"`{name}` must be a dictionary or JSON string: got {original_type}")
+        raise TypeError(f"`{name}` must be a dictionary or JSON string: gottt {original_type}")
 
     data_json = json.dumps(maybe_dict)
 
     if size_limit_bytes is not None:
-        size_bytes = sys.getsizeof(data_json)
+        size_bytes = len(data_json)
```

Comment:
Made this change just to be consistent with the recent back-end change. `sys.getsizeof(data_json))` gets us the number of bytes in RAM, which is not what we want. We want to say the limit is 1024 bytes after base64 encoding.